crates/dprint/src/cli/run_cli.rs

```diff
@@ -249,6 +250,45 @@ fn output_editor_info<TEnvironment: Environment>(
     Ok(())
 }
 
+fn build_glob_set<TEnvironment: Environment>(
+    config: &ResolvedConfig,
+    environment: &TEnvironment,
+    file_patterns: &mut Vec<String>,
+) -> Result<GlobSet, ErrBox> {
+    let mut builder = GlobSetBuilder::new();
+    let cwd = environment.cwd()?;
+    let config_base_path = config.base_path.to_string_lossy();
+    let base_path = match config_base_path.as_ref() {
+        "./" => cwd.to_string_lossy(),
```

Comment:
It seems that `base_path` will return the absolute path if you're not currently in that as the working directory, but otherwise return `./`. 

Not sure if this is the best way, or if `base_path` should just be updated to always have the absolute path.